app/javascript/components/util/DeleteDialog.js

```diff
@@ -78,6 +81,26 @@ class DeleteDialog extends React.Component {
               <div className="character-limit-text">{MAX_REASON_LENGTH - this.state.delete_reason_text.length} characters remaining</div>
             </React.Fragment>
           )}
+          {this.props.type === 'Lab Result' && this.props.promptSymptomOnset && (
+            <div>
+              <p>
+                Since this record does not have a Symptom Onset Date, deleting this positive lab result may result in the record not ever being eligible to
+                appear on the Records Requiring Review line list. Please enter a symptom onset date to prevent this from happening:
+              </p>
+              <Form.Label className="nav-input-label">SYMPTOM ONSET</Form.Label>
```

Comment:
same comment as in the lab modal

Reply:
https://mitrecorp.slack.com/archives/C01A6P94R9B/p1623861467059600?thread_ts=1623785133.027100&cid=C01A6P94R9B